automol/vmat.py

```diff
@@ -66,36 +76,33 @@ def from_data(symbs: tuple[str, ...], key_mat, name_mat=None, one_indexed=None):
 
 # # V-Matrix/V-Matrix common functions (document these as z-matrix functions)
 # # # getters
-def symbols(vma, idxs: list[int] | None = None) -> list[str]:
+def symbols(vma: VMatrix, idxs: list[int] | None = None) -> list[str]:
     """Obtain the atomic symbols for all atoms defined in the V-Matrix.
 
     :param vma: V-Matrix
-    :type vma: automol V-Matrix data structure
     :param idxs: indices of atoms to obtain information for
-    :type idxs: List[int]
-    :returns: The list of atomic symbols
-    :rtype: List[str]
+    :return: The list of atomic symbols
     """
     if vma:
-        symbs, *_ = tuple(zip(*vma))
+        symbs, *_ = tuple(zip(*vma, strict=False))
```

Comment:
I would prefer to change these all to `strict=True`, since we are always expecting the zip arguments to have the same length (if they aren't, it's probably a bug).